assets/script.js

```diff
@@ -144,7 +127,9 @@ function nextQuestion() {
         createQuestions();
     } // Should send user to scoreboard after clicking answer on final question. For some reason it's taking 2 clicks. Need to figure it out.
     if (questionNumber === questions.length) {
-        showScoreboard();}
+        showScoreboard();
+        stopTimer();
```

Comment:
not stopping the timer here is what was causing the "reset game" button bug. the timer was still counting down and triggering the `loseGame` function in the background after the game was over.